react/src/components/UserResourcePolicyList.tsx

```diff
@@ -187,9 +187,16 @@ const UserResourcePolicyList: React.FC<UserResourcePolicyListProps> = () => {
                   variables: {
                     name: row.name,
                   },
-                  onCompleted: (data, errors) => {
-                    if (errors) {
-                      message.error(errors[0]?.message);
+                  onCompleted: (res, errors) => {
+                    if (!res?.delete_user_resource_policy?.ok) {
+                      message.error(res?.delete_user_resource_policy?.msg);
+                      return;
```

Comment:
```suggestion
                      message.error(res?.delete_user_resource_policy?.msg);
                      setInFlightResourcePolicyName('');
                      return;
```

Wouldn't it be nice to turn off the loading indicator?

Reply:
I resolved it by adding a flight checking in loading.